packages/server/server.js

```diff
@@ -0,0 +1,27 @@
+const express = require('express')
+const session = require('express-session')
+
+// saves express function to the variable app
+const app = express()
+
+// sets up boilerplate routes to access the database
+app.use('/api', getRoutes)
+app.use('/api/post', postRoutes)
+app.use('/api/auth', authRoutes)
```

Comment:
I wonder if it would be more worth while to have a root router.
```js
app.use('/api', apiRoutes);
```
```js
// apiRoutes.js
const router = require('express').Router();

router.use('/products', require('./products'));
router.use('/categories', require('./categories'));

module.exports = router;
```

Reply:
It's also worth noting that if we were to merge this it'd break `main`. We want to avoid merging broken code.

Reply:
Thank you for the feedback! Will fix changes.